pkger/models.go

```diff
@@ -107,7 +112,11 @@ func (k Kind) ResourceType() influxdb.ResourceType {
 }
 
 func (k Kind) title() string {
-	return strings.Title(k.String())
+	pieces := strings.Split(string(k), "_")
+	for i := range pieces {
+		pieces[i] = strings.Title(pieces[i])
```

Comment:
heh, didn't know this was in strings

Reply:
`strings` actually has a ton of useful funcs. Has stuff for `map`, 'trim`ing, and a boat load of other goodies that come up quite often